### PR TITLE
NFS: Added an example to consume the nfs export created by nfs operator.

### PR DESCRIPTION
**Pull request by rohan47**
Signed-off-by: rohan47 <rohgupta@redhat.com>

**Description of your changes:**
NFS: Added an example to consume the nfs export created by nfs operator.

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**

- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/master/development-flow.html#submitting-a-pull-request)
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.
- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.
- [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
- [ ] Code generation (`make codegen`) has been run to update object specifications, if necessary.
- [ ] Comments have been added or updated based on the standards set in [CONTRIBUTING.md](https://github.com/rook/rook/blob/master/CONTRIBUTING.md#comments)
- [ ] Add the flag for skipping the CI if this PR does not require a build. See [here](https://github.com/rook/rook/blob/master/INSTALL.md#skip-ci) for more details.

[skip ci]

### CODE REVIEW

File: Documentation/nfs.md [hunk 1]
```diff
@@ -165,6 +165,48 @@ spec:
...
+1. A writer pod that will write random data to the NFS share so the website will continually update
+
+Start both the busybox pod (writer) and the web server from the `cluster/examples/kubernetes/nfs` folder:
+
+```console
+kubectl create -f busybox-rc.yaml
+kubectl create -f web-rc.yaml
+```
+
+Let's confirm that the expected busybox writer pods and web server pods are **all** up and in the `Running` state:
```
Comment (leseb) [inline Documentation/nfs.md]:
```suggestion
Let's confirm that the expected busybox writer pod and web server pod are **all** up and in the `Running` state:
```

File: Documentation/nfs.md [hunk 2]
```diff
@@ -165,6 +165,48 @@ spec:
...
+```
+
+In order to be able to reach the web server over the network, let's create a service for it:
+
+```console
+kubectl create -f web-service.yaml
+```
+
+We can then use the busybox writer pod we launched before to check that nginx is serving the data appropriately.
+In the below 1-liner command, we use `kubectl exec` to run a command in the busybox writer pod that uses `wget` to retrieve the web page that the web server pod is hosting.  As the busybox writer pod continues to write a new timestamp, we should see the returned output also update every ~10 seconds or so.
```
Comment (leseb) [inline Documentation/nfs.md]:
```suggestion
In the below 1-liner command, we use `kubectl exec` to run a command in the busybox writer pod that uses `wget` to retrieve the web page that the web server pod is hosting. As the busybox writer pod continues to write a new timestamp, we should see the returned output also update every ~10 seconds or so.
```

Comment (rohan47) [reply Documentation/nfs.md]: done

File: Documentation/nfs.md (hunk 1 above)
Comment (rohan47) [reply Documentation/nfs.md]: done

**Pull request merged**